boards/laird_connect/bl654_dvk/bl654_dvk_nrf52840_pa.dts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2019 Laird Connectivity
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bl654_dvk.dts"
+
+/* SPDX-License-Identifier: Apache-2.0 */
+
+&uart0 {
+	current-speed = <1000000>;
```

Comment:
??

Reply:
Will remove the extra SPDX header.

I'll also remove the speed adjustment. It's better for compatibility out-of-the-box, right?